src/seedu/addressbook/data/AddressBook.java

```diff
@@ -22,13 +24,16 @@
 
     private final UniquePersonList allPersons;
     private final UniqueTagList allTags; // can contain tags not attached to any person
+    
+    private final ArrayList<Tagging> taggingList;
```

Comment:
we discussed in class about the benefits of putting the list here and the difference in the two diagrams.